EDA/KDWD_EDA.ipynb

```diff
@@ -1665,7 +1665,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.7.4"
```

Comment:
Was this Python version downgrade intentional? I expect this would break a lot of dependencies

Reply:
This was Hardik's notebook. But it isn't intentional I think. Both versions would work. There aren't many dependencies on our repo right now.